docs/basics/networkvariable.md

```diff
@@ -546,136 +567,17 @@ The above code could be tested by:
-        // Get the string size in bytes
-        reader.ReadValueSafe(out stringSize);
-
-        // If there is nothing, then we are done
-        if (stringSize == 0)
-        {
-            return;
-        }
-
-        // allocate an byte array to 
-        var byteArray = new byte[stringSize];
-        var tempByte = (byte)0;
-        for(int i = 0; i < stringSize; i++)
-        {
-            reader.ReadValueSafe(out tempByte);
-            byteArray[i] = tempByte;
-        }
-        
-        // Convert it back to a string
-        Text = System.Text.Encoding.ASCII.GetString(byteArray);
-    }
-
-    public override void ReadDelta(FastBufferReader reader, bool keepDirtyDelta)
-    {
-        // Do nothing for this example
-    }
-
-    public override void WriteDelta(FastBufferWriter writer)
-    {
-        // Do nothing for this example
-    }
-}
-```
-
-For simplicity purposes, the above example doesn't handle updating connected clients with any changes to the `Text` string property. It is just one possible approach you might take if you are not concerned about the memory allocation of the temporary `byte` array. It is a "bare minimum and non-optimized" example in order to demonstrate that as long as your serialization process knows what to write and what to read you can serialize any form of managed type.
-
-:::important
-If you already had a maximum string size in mind, you could pre-allocate the byte array to avoid the cost of memory allocation.  The downside to this is that you would lose the "managed" flexibility of being able to handle varying message sizes, but the upside (as you will find out in the next example) is that you would always only send the exact number of bytes the string consumes and not send the entire pre-allocated buffer (i.e. you save on bandwidth). 
+`NetworkVariable` does not support being nested inside of other `NetworkVariable` classes. Netcode for GameObjects does a code generation step to define serialization callbacks for each type it finds used in a `NetworkVariable`, and it looks for those variables as fields of `NetworkBehaviour` types - a `NetworkVariable` that's declared anywhere else will be missed by the sweep. Instead, declare `NetworkVariable` or `NetworkList` properties within the same `NetworkBehaviour` that you have declared your custom `NetworkVariableBase` implementation within.
 :::
 
-However, if you already know the maximum size of the `string` that you want to synchronize then there is another way to handle synchronizing "fixed" strings.
+## Strings
 
-### Strings: NetworkVariable FixedString Example
+While NetworkVariable does support managed `INetworkSerializable` types, strings are not in the list of supported types. This is because strings in C# are immutable types, preventing them from being deserialized in-place, so every update to a `NetworkVariable<string>` would cause a GC allocation to create the new string, which may lead to performance problems.
```

Comment:
What's a GC allocation? Is GC an acronym for "game client"?

Reply:
Garbage Collection.